### DISCUSSION TITLE
Version Control and Rollback for configuration files

### DISCUSSION THREAD

**Discussion by breakenknife** [0xJacky/nginx-ui#597] [Ideas]
Requirements:
1. Display the modification history of files.
2. Ability to roll back to a specified version.
3. Ability to compare different versions.

---
需求:
1.  可显示文件的修改记录.
2. 可回滚到指定的版本.
3. 可对比不同版本.

**Comment by jearton** [comment]
+1

**Comment by 0xJacky** [comment]
+2

**Comment by kekylin** [comment]
+3
未来可期

**Comment by zifsy** [comment]
+4

**Comment by BulgarianHealer** [comment]
+5

**Comment by goshku** [comment]
+6

**Comment by hongmd** [comment]
+7
